imports/client/components/AudioConfig.tsx

```diff
@@ -262,7 +262,6 @@ const AudioConfig = () => {
       <audio
         ref={audioRef}
         autoPlay
-        playsInline
```

Comment:
I slipped this (and the matching change in `CallSection`) into this PR since I saw it's what's causing #1029
to fail presubmit checks at https://github.com/deathandmayhem/jolly-roger/runs/8193203773?check_suite_focus=true and this change rewrites the line in `CallSection` anyway.

The backstory is kinda fun -- this is a vestige of the distant past from when I was bringing up the mesh WebRTC implementation with video because I didn't yet have anything like a spectrogram and I was sick of hearing feedback when the connection was successfully established.  Once I had the implementation working reliably, I switched the `video` element to an `audio` one, without realizing this property isn't valid for `audio`.